### PR TITLE
[Node Manager] Add & Update Unit Test Cases

### PR DESCRIPTION
**Pull request by kimeunju108**
This PR  shows the following changes:
* Added more UTs
* Modified UTs from controller test to controller and service test

### CODE REVIEW

File: scripts/build_coverage.sh [hunk 1]
```diff
@@ -8,7 +8,10 @@ mvn compile
 cd ..
 cd ./web
 mvn compile
+<<<<<<< HEAD
```
Comment (xieus) [inline scripts/build_coverage.sh:11]: Merge conflict.

Comment (kimeunju108) [reply scripts/build_coverage.sh]: This conflict was fixed and pushed to repository again.

File: scripts/build_coverage.sh [hunk 2]
```diff
@@ -8,7 +8,6 @@ mvn compile
 cd ..
 cd ./web
 mvn compile
-cd ..
```
Comment (xieus) [inline scripts/build_coverage.sh:11]: Please add this line back. Revoking this fix will bring the same problem that we hit before PR #247 was merged.

Comment (kimeunju108) [reply scripts/build_coverage.sh]: This mistake is fixed now.

**Pull request merged**